### PR TITLE
Added ability to install Verovio with Cocoapods.

### PR DESCRIPTION
**Pull request by vitalii-unicsoft**

### CODE REVIEW

File: verovio.podspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	s.swift_versions = ['5.1']
+  s.source_files   = 'src/**/*.{h,cpp,cc}', 
+									   'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}',
+										 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h,cpp}'
+	s.public_header_files = 'src/**/*.{h}', 
+											    'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}',
+													'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h}'
+	s.platform       = :ios, '12.0'
+  s.resources = 'data'
+	s.xcconfig = {
```
Comment (craigsapp) [inline verovio.podspec]: Indentation is no so great.

File: verovio.podspec [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  s.summary        = 'Verovio'
+  s.source         = { :git => 'https://github.com/vitalii-unicsoft/verovio', :tag => 'version-' + s.version.to_s }
+	s.swift_versions = ['5.1']
+  s.source_files   = 'src/**/*.{h,cpp,cc}', 
+									   'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}',
+										 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h,cpp}'
+	s.public_header_files = 'src/**/*.{h}', 
+											    'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}',
+													'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h}'
+	s.platform       = :ios, '12.0'
```
[the remark this answers is not in the document]
Comment (vitalii-unicsoft) [reply verovio.podspec]: In the original Xcodeproj it is set to 12.1

[the remark this answers is not in the document]
Comment (Noroxs) [reply verovio.podspec]: You are right. I think it's fine to leave it like this.

File: verovio.podspec [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  s.name           = 'Verovio'
+  s.version        = '2.1.0'
+  s.license        = { :type => 'LGPL' }
+  s.homepage       = 'https://www.verovio.org/index.xhtml'
+  s.authors        = { 'Contributors List' => 'https://github.com/rism-ch/verovio/graphs/contributors' }
+  s.summary        = 'Verovio'
+  s.source         = { :git => 'https://github.com/rism-ch/verovio.git', :tag => 'version-' + s.version.to_s }
+  s.swift_versions = ['5.1']
+  s.source_files   = 'src/**/*.{h,cpp,cc}', 'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}', 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h,cpp}'
+  s.public_header_files = 'src/**/*.{h}', 'include/{hum,json,midi,pugi,utf8,vrv}/*.{h,hpp}', 'libmei/{attc*,atts_a*,atts_c*,atts_ex*,atts_fa*,atts_g*,atts_m*,atts_n*,atts_pa*,atts_s*,atts_v*,attt*}.{h}'
```
Comment (lpugin) [inline verovio.podspec]: Please remove the `*` on the libmei files and have full names instead

Comment (vitalii-unicsoft) [reply verovio.podspec]: Done

File: verovio.podspec (hunk 1 above)
Comment (vitalii-unicsoft) [reply verovio.podspec]: fixed

File: verovio.podspec [hunk 4]
```diff
@@ -0,0 +1,35 @@
+Pod::Spec.new do |s|
+  s.name           = 'Verovio'
+  s.version        = '2.1.0'
+  s.license        = { :type => 'BSD' }
+  s.homepage       = 'https://github.com/vitalii-unicsoft/verovio'
```
[the remark this answers is not in the document]
Comment (vitalii-unicsoft) [reply verovio.podspec]: done

File: verovio.podspec [hunk 5]
```diff
@@ -0,0 +1,35 @@
+Pod::Spec.new do |s|
+  s.name           = 'Verovio'
+  s.version        = '2.1.0'
+  s.license        = { :type => 'BSD' }
```
[the remark this answers is not in the document]
Comment (vitalii-unicsoft) [reply verovio.podspec]: Done

**Pull request merged**